libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/compound/LightDesignTokens.kt

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.designsystem.theme.compound
+
+import androidx.compose.ui.graphics.Color
+
```

Comment:
Is it easy enought to add a comment here that explains these are autogenerated files?